adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastqSuite.scala

```diff
@@ -0,0 +1,63 @@
+package org.bdgenomics.adam.cli
+
+import java.io.File
```

Comment:
Nit: Can you move this to the end of the imports? We're trying to keep imports alphabetized.